src/redux/categories/tests/categories.reducer.test.js

```diff
@@ -0,0 +1,36 @@
+import categoriesReducer from '../categories.reducer';
+import { SET_CATEGORIES } from '../categories.types';
+
+describe('categories.reducer.test', () => {
+  let initialState;
+  beforeEach(() => {
+    initialState = {
+      list: [],
+      loading: true
+    };
+  });
+
+  it('should return default state', () => {
+    expect(categoriesReducer(initialState, {})).toEqual(initialState);
+  });
+
+  it('should return state with new categories', () => {
+    const newCategories = {
```

Comment:
Please, create actions via action creators because frankly speaking reducer waits for the object created by the action creator

Reply:
Created with action creaters